src/main/java/racingcar/domain/Moving.java

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain;
+
+import java.util.Random;
+
+public enum Moving {
+    GO, STOP;
+
+    private static final Random random = new Random();
```

Comment:
상수이니 컨벤션에 맞춰 대문자로 작성하시면 어떨까요?
```suggestion
    private static final Random RANDOM = new Random();
```

Reply:
요것도 다음 스텝 진행할 때 반영할께요 감사합니다!